.github/actions/publish-images/action.yml

```diff
@@ -64,9 +64,20 @@ runs:
             IMG_NAME="$(getVarValue "${PREFIX}" _IMG)"
             echo "::group::Push image $IMG_NAME $IMG_TAG"
             GHCR_PATH="ghcr.io/${{ github.repository }}/${IMG_NAME}"
+
             if [ "$(imageTagExists "$IMG_NAME" "$IMG_TAG")" == "200" ]; then
               echo "Image already exists: $IMG_NAME:$IMG_TAG -- not overwriting"
               echo "* ($IMG_NAME:$IMG_TAG -- already exists, not overwriting)" >> "$GITHUB_STEP_SUMMARY"
```

Comment:
Hi @brostk , I was just checking in on the status of this "scheduled SecRel" fix, and noticed that your PR needed another review.  So taking a look at it, it looks good, but the one thing I noticed was that we are still writing this "already exists, not overwriting" message to  stdout, even though that won't be true for the scheduled event trigger. If this sounds correct, I wanted to recommend that we skip these echo statements to avoid misleading messages in the build log. I would imagine that adding an else condition to the if on line 72 would be an easy way to remove those "not overwritting" messages.

Reply:
@nelsestu Thanks for the second set of eyes - I thought it best to keep the echo since `docker tag ...; docker push ...` isn't being run and so we aren't actually overwriting the image in this case, we're just running the scan on it.  Does that sound right to you?  I'm happy to move the echo around so it's more representative if you think leaving it as-is might be misleading.

Reply:
Ah! that makes sense and would be correct but the non-obvious implication of running a scan is that if that scan succeeds then it will result in the republication of images.  It is true that we won't change the tags and it is true that this action won't be pushing any images, however once SecRel passes it will sign the images and republish those.  This actually touches on one point of feedback we gave to the SecRel team last week, which was that it would be great if there was some way to query ghcr.io inquiring whether an image has been SecRel signed. At the present moment, and given how we tag images today, we don't know whether an image has been SecRel signed or not.  Going back to the GH Action logs is one way of knowing.  Pulling the image and then checking the binary for signatures would be another.